<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QMainWindow" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1553</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginScreen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="toggleFullscreenButton">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ðŸ—–</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Bookcave</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>560</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_field">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>259</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>259</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>470</y>
      <width>259</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_field">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>259</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
